static/app/views/alerts/rules/issue/addIntegrationRow.tsx

```diff
@@ -0,0 +1,116 @@
+import {useCallback, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import PluginIcon from 'sentry/plugins/components/pluginIcon';
+import {space} from 'sentry/styles/space';
+import type {IntegrationProvider, Project} from 'sentry/types';
+import type {Organization} from 'sentry/types/organization';
+import useApi from 'sentry/utils/useApi';
+import {AddIntegrationButton} from 'sentry/views/settings/organizationIntegrations/addIntegrationButton';
+
+type Props = {
+  closeModal: () => void;
+  integrationSlug: string;
+  organization: Organization;
+  project: Project;
+  setHasError: (boolean) => void;
+};
+
+function AddIntegrationRow({
+  integrationSlug,
+  organization,
+  project,
+  closeModal,
+  setHasError,
+}: Props) {
+  const [provider, setProvider] = useState<IntegrationProvider | null>(null);
+
+  const api = useApi();
+  const fetchData = useCallback(() => {
+    if (!integrationSlug) {
+      return Promise.resolve();
+    }
+
+    const endpoint = `/organizations/${organization.slug}/config/integrations/?provider_key=${integrationSlug}`;
+    return api
+      .requestPromise(endpoint)
+      .then(integrations => {
+        setProvider(integrations.providers[0]);
+      })
+      .catch(error => {
+        console.error('Error fetching data:', error);
+        setHasError(true);
+      });
+  }, [integrationSlug, api, organization.slug]);
+
+  useEffect(() => {
+    fetchData();
+  }, [fetchData]);
+
```

Comment:
You can add `console.log("provider", integrations)` here or any other log you like to see why it's not loading. They will show up in the terminal when you run the test.